css/oraui.css

```diff
@@ -399,4 +399,21 @@ input:checked+.slider:before {
     margin-left: .5vh;
     position: absolute;
     right: 0;
-}
\ No newline at end of file
+}
+.ent-image{
+  padding-top:10%;
+  margin: 1% 1% 1% 1%;
```

Comment:
you can just write it as `margin: 1%;`

Reply:
sorry i experimented alot of different ones then i just got fed up and kept all 1% hahaha, forgot to do that